api/live.js

```diff
@@ -217,6 +224,26 @@ async function handleEmailContrib(userEmail, req, res) {
 
 async function handleCheckin(userId, req, res) {
   try {
+    const profile = await models.HackerProfile.findOne({
+      where: { userId: userId }
+    });
+    const profileStatus = profile.get("status");
+    const invalidStatuses = [
```

Comment:
Do we have an enum statuses somewhere? This many magic strings scare me

Reply:
we don't, prob one of the regrets i have is not having some constant file early on, tho i can address in a follow up